### DISCUSSION TITLE
Custom completions for PowerShell

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [PowerShell/PowerShell#16488] [Q&A]
Hello! Where I have to suggest my custom completions for any command in PowerShell? To take a long story short I wanna write `mono` completions like [these ones](https://github.com/fish-shell/fish-shell/pull/8452) because they are missing now.

**Comment by iSazonov** [comment]
@EmilySeville7cfg You could look the example https://github.com/PowerShell/Modules/tree/master/Modules/Microsoft.PowerShell.UnixCompleters
